{"frames":[{"func":"bind_rows(df_sum1, df_sum2, df_sum6, df_sum10, df_sum14, df_sum16, \n    df_sum21, df_sum27, df_sum28, df_sum33, df_sum35, df_sum38, \n    df_sum46, df_sum48, df_sum50, df_sum52, df_sum64, df_sum70, \n    df_sum74, df_sum77, df_sum81, df_sum84, df_sum99, df_sum107,  ...","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"list2(...)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in list2(...) : object 'df_sum248' not found\n"}